chore(docs): improve openTelemetry installation command (#1797)
kumahq/kuma-website: app/_src/guides/otel-metrics.md

    containerPort: 8889
    servicePort: 8889
    protocol: TCP
image:
  repository: 'otel/opentelemetry-collector-contrib'
" > values-otel.yaml
```

This is the Helm chart configuration we will be using. This will configure OpenTelemetry collector to listen on grpc port `4317` for metrics 
pushed by dataplane proxy, process and expose collected metrics in Prometheus format on port `8889`. In the next step we 

> will configure Prometheus to scrape these metrics. Our configuration relies on [the contrib distribution](https://github.com/open-telemetry/opentelemetry-collector-contrib#opentelemetry-collector-contrib) of opentelemetry-collector so we set this in the values.

Reviewer: ```suggestion
will configure Prometheus to scrape these metrics. Our configuration relies on [the contrib distribution](https://github.com/open-telemetry/opentelemetry-collector-contrib) of opentelemetry-collector so we set this in the values.
```

> will configure Prometheus to scrape these metrics. Our configuration relies on [the contrib distribution](https://github.com/open-telemetry/opentelemetry-collector-contrib?tab=readme-ov-file#opentelemetry-collector-contrib) of opentelemetry-collector so we set this in the values.

Reviewer: ```suggestion
will configure Prometheus to scrape these metrics. Our configuration relies on [the contrib distribution](https://github.com/open-telemetry/opentelemetry-collector-contrib#opentelemetry-collector-contrib) of opentelemetry-collector so we set this in the values.
```

    containerPort: 8889
    servicePort: 8889
    protocol: TCP
image:
  repository: 'otel/opentelemetry-collector-contrib'
" > values-otel.yaml
```

This is the Helm chart configuration we will be using. This will configure OpenTelemetry collector to listen on grpc port `4317` for metrics 
pushed by dataplane proxy, process and expose collected metrics in Prometheus format on port `8889`. In the next step we 

> will configure Prometheus to scrape these metrics. Refer to openTelemetry-collector helm installation [UPGRADING](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/UPGRADING.md#0880-to-0890),
> it requires the parameter `image.repository` must be set and our current configurations rely on `Contrib distribution` image repository.

Reviewer: ```suggestion
will configure Prometheus to scrape these metrics. Our configuration relies on [the contrib distribution](https://github.com/open-telemetry/opentelemetry-collector-contrib?tab=readme-ov-file#opentelemetry-collector-contrib) of opentelemetry-collector so we set this in the values.
```

```shell
helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts

> helm install --namespace mesh-observability opentelemetry-collector open-telemetry/opentelemetry-collector -f values-otel.yaml --set image.repository=otel/opentelemetry-collector-contrib

Reviewer: Hmm I think this needs an explanation, why are we setting this image?

Author: ok, I gonna supplement it.
